system/security/digisign/digisign.cpp

```diff
@@ -0,0 +1,270 @@
+/*##############################################################################
+
+    HPCC SYSTEMS software Copyright (C) 2017 HPCC Systems®.
+
+    Licensed under the Apache License, Version 2.0 (the "License");
+    you may not use this file except in compliance with the License.
+    You may obtain a copy of the License at
+
+       http://www.apache.org/licenses/LICENSE-2.0
+
+    Unless required by applicable law or agreed to in writing, software
+    distributed under the License is distributed on an "AS IS" BASIS,
+    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+    See the License for the specific language governing permissions and
+    limitations under the License.
+############################################################################## */
+#include "jliball.hpp"
+#ifdef _USE_ZLIB
+#include <openssl/pem.h>
+#include <openssl/err.h>
```

Comment:
Should you add a few more includes ?  Such as:
```
include <openssl/rsa.h>
include <openssl/crypto.h>
include <openssl/ssl.h>
```

Reply:
Not sure why I would include them if I am able to build?   Perhaps pem.h does the include of those files?